rpa-home/modules/ROOT/pages/automation-security-securesessions.adoc

```diff
@@ -0,0 +1,50 @@
+
+Or you could use what we call _Secure Sessions_.
+
+Secure Sessions are hidden Desktop Sessions. They run in the background and only the bot can access them. An additional advantage of the Secure Sessions is, that the bot can multiply itsself to run several process instances in parallel.
+
+To use Secure Sessions you must ensure that all prerequisites are met and all responsible persons are informed.
+
+== General Considerations
+
+* At the same time, a bot can only run in Desktop or in Secure Session mode.
+* You can not switch the session mode while configurations are deployed on the bot.
+* Run configurations specify a session mode, too.
+* You can only deploy configurations to a bot that is in the same session mode.
+* You can specify the login data for the bot and the maximum number of Secure Sessions that the bot may run in parallel in the RPA Bot Configurator or in the RPA Manager.
+** The maximum number of Secure Sessions set and enabled in the RPA Bot Configurator overwrites the same setting of RPA Manager.
+* Desktop Sessions:
+** The specified user must be logged in to this session to deploy a run configuration. 
+** Do not close or minimize the console session while the bot executes a process.
+* Secure Sessions:
+** The higher the number of secure sessions the lower the performance of the bot. Too much load can cause process slowdown or disruptions. Check the system data and the system requirements documentation to determine the maximum number of sessions that the bot can safely execute in parallel.
+** Before deploying a run configuration on a bot that has only one secure session licensed, do not disconnect the session, but sign out the specified user. 
+** Do not sign in as the specified user to a computer where a bot with one licensed secure session is running a process.
+
+== Next Steps
+
+. In the documentation, check the System Requirements.
+. On the computer where you want to run automations and where the bot therefore is installed, check the system data:
++
+* Operating system
+* RAM
+* Terminal Server software
+* Login data for the bot
+* RPA Bot Configurator settings
+. If you want to use default session settings for the run configuration, set the bot login data and the maximum number of parallel Secure Sessions in the Bot Configurator. 
+. In RPA Manager, enter the relevant data or a contact person in the description of the bot in the Bots view of the Bot Management module.
+.. If the bot should use Secure Sessions and the RPA Bot Configurator does not contain this setting, set the maximum number of parallel Secure Sessions that the bot may use via the Secure Session Assignment view.
```

Comment:
```suggestion
.. If the bot should use Secure Sessions and the RPA Bot Configurator does not contain this setting, set the maximum number of parallel Secure Sessions that the bot can use via the Secure Session Assignment view.
```

Reply:
I noticed that you had a single sub-step. This should just be a hanging paragraph. Our style is to not have single un/ordered list items.